server/models/Room.js

```diff
@@ -0,0 +1,40 @@
+const { Schema, model, trusted } = require('mongoose');
+
+const roomSchema = new Schema(
```

Comment:
The schema should be used or exported for later use.  Since I know the resolvers haven't been created it won't affect anything right now, but leaving a comment here as a reminder that this file will not function without some additional stuff.

Reply:
- Added export for roomschema